projects/storefrontlib/src/shared/components/modal/modal.service.ts

```diff
@@ -18,10 +18,17 @@ export class ModalService {
 
     activeModal = this.ngbModalService.open(content, options);
     this.modals.push(activeModal);
+    this.handleModalRemoveEvents(activeModal);
```

Comment:
Would you mind adding some unit tests for this service?

Reply:
Added